pom.xml

```diff
@@ -522,6 +522,7 @@
                 change to fix a integer overflow issue.
                 TODO: remove this after Parquet 1.13 release -->
               <exclude>org.apache.parquet.column.values.dictionary.DictionaryValuesWriter#dictionaryByteSize</exclude>
+              <exclude>org.apache.parquet.hadoop.util</exclude>
```

Comment:
With this exclusion we won't check API compatibility for this package. If want to do this we should state somewhere that we don't guarantee backward compatibility of the classes in this package. We've had bad times doing things like this. Don't we have other solutions to keep backward compatibility instead?

Reply:
Let me narrow it down to the impacted classes only.

Reply:
Maybe I was misleading. For me it doesn't matter if we have a package or classes listed here. My problem is that we are breaking backward compatibility from the API point of view. If we think it is OK to break the compatibility of this package then I am fine having it here but with a separate comment. (For `DictionaryValuesWriter` we did not have a choice to not to break the API that's why the TODO is there.)
So, if the `util` package is not for public use than we shall add `@Private` to all of its classes and maybe also add a `package-info.java` stating that this is not for public use and add similar comments here as well. Otherwise, we should maintain backward compatibility for these classes as well.